src/app/index.js

```diff
@@ -141,10 +141,18 @@ class App extends React.Component {
    */
   getUserProfile () {
     let self = this;
-    let url = `${self.state.base}${self.state.user.path}`;
-    return axios.get(url).then(res => {
-      self.setState({user: res.data});
-    });
+    // let url = `${self.state.base}${self.state.user.path}`;
+    let url = `/api/user-profile`;
```

Comment:
does this need to be set back to the uncommented state?

Reply:
removing in the next PR